etc/WMAgentConfig.py

```diff
@@ -61,9 +61,13 @@
 globalDBSUrl = "https://cmsweb.cern.ch/dbs/prod/global/DBSReader"
 globalDBSVersion = "DBS_2_0_8"
 
-# Job retry information.  This includes the number of times a job will tried and
-# how long it will sit in cool off.
-maxJobRetries = 3
+# List of SE for T1 _Disk endpoints (TODO clean this up at some point)
+diskSites = ['storm-fe-cms.cr.cnaf.infn.it','srm-cms-disk.gridpp.rl.ac.uk',
```

Comment:
Make sure @alexanderrichards knows about this and can adjust in what he's doing.